src/cli/commandGenerateConfiguration.ts

```diff
@@ -108,30 +126,79 @@ export async function commandGenerateConfiguration(argv: string[]) {
         return
     }
 
-    const { tsconfig, importPath, index = findDefaultIndex(importPath) } = args
+    if (!["typescript", "plain", "flow"].includes(args.mode)) {
+        printInvalidMode(args.mode)
+        return
+    }
 
-    if (!index || index.length == 0) {
-        printInvalidIndex()
+    let components
+    let result
```

Comment:
Will probably need to import the type as well:

```suggestion
    let result: EmitResult[]
```